listener.py

```diff
@@ -86,19 +84,18 @@ def process_request(self, connection_socket):
             if not (full_msg[0] == "{"):
                 full_msg = full_msg[2::]
         except IndexError as error:
-            logger.error(error)
+            log_error(error)
```

Comment:
Going forward we should try to use global_logger.log_error rather than global_logger.logger.error because it provide much more useful information about the errors.